.travis.yml

```diff
@@ -1,11 +1,11 @@
 language: node_js
-
 node_js:
   - 0.10
-  - 0.11
 
 before_script:
-  - npm install -g gulp
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
```

Comment:
@jlukic do you have an SauceLabs or BrowserStack account we could borrow for browsers?